src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(double radius) {
+        this.type = getClass().getSimpleName();
+        this.radius = radius;
+        this.color = ColorSupplier.getRandomColor();
```

Comment:
It's bad to create a figure immediately with a random color, you need to pass it to the constructor.
What if we need a shape with a fixed color? Will it be necessary to create several shapes until the necessary one is created? :)